test-case-reporting/src/test_result_record.ts

```diff
@@ -120,89 +123,109 @@ export class TestResultRecord {
     await this.db('test_runs').insert(testRuns);
   }
 
-  /**
-   * Gets a DB.Build from the database by its build number.
-   * If there is no such build, it returns undefined.
-   * @param buildNumber The number of the Travis build, as a string.
-   */
-  private async getDbBuildFromBuildNumber(
-    buildNumber: string
-  ): Promise<DB.Build> {
-    return this.db<DB.Build>('builds')
-      .where('build_number', buildNumber)
-      .first();
-  }
+  /* eslint @typescript-eslint/camelcase: "off" */
```

Comment:
Don't forget to re-enable this after the function

Reply:
When I re-enable it after the function, it complains about the snake case within the function -- maybe there's something broken in my package config or linter config? I have no clue -- I didn't touch any of it.

Reply:
Fixed.